vulnerabilities/api.py

```diff
@@ -46,11 +46,29 @@ class MinimalPackageSerializer(serializers.HyperlinkedModelSerializer):
     Used for nesting inside vulnerability focused APIs.
     """
 
+    def get_affected_vulnerabilities(self, package):
+        parent_affected_vulnerabilities = package.fixed_package_details.get("vulnerabilities") or []
+        affected_vulnerabilities = []
+
+        for vuln in parent_affected_vulnerabilities:
+            self.get_vulnerability(vuln, affected_vulnerabilities)
+
+        return affected_vulnerabilities
+
+    def get_vulnerability(self, vuln, affected_vulnerabilities):
+        affected_vulnerability = {}
+
+        if vuln.get("vulnerability"):
```

Comment:
```suggestion
        vulnerability = vuln.get("vulnerability"):
```

Reply:
Thanks @TG1999 I'll address these asap and hopefully we can put this PR to bed -- though still awaiting comments from @pombredanne .  ;-)

Reply:
Done.